docker/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM php:7.0-fpm-alpine
```

Comment:
I had big issues when using varnish with alpine don't know if it's still the case, but compilation of the vcl was hard on alpine due to the musl lib